sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -334,12 +334,10 @@ private void addProdCredentials(List<TokenCredential> credentials) {
     // Helper to add dev credentials
     private void addDevCredentials(List<TokenCredential> credentials) {
         credentials.add(new IntelliJCredential(tenantId, identityClientOptions.clone()));
+        credentials.add(new VisualStudioCodeCredential(tenantId, identityClientOptions.clone()));
```

Comment:
Do we need a changelog entry regarding adding VSCodeCredential to DAC?

Reply:
Yes, we do

Reply:
For Java, its already added by the PR that added the feature.